### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
# Closes NaN

## What's Changed

- Updated the cookie based on the latest template.
- Updated Nautobot to `v1.6.0`.
- Updated pylint to `v2.17.7`.
- Fixed pylint issues.

## Drift Manager

Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-welcome-wizard.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-welcome-wizard",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-welcome-wizard",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@smith-ntc",
        "full_name": "Network to Code, LLC",
        "email": "info@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "welcome_wizard",
        "verbose_name": "Welcome Wizard",
        "plugin_slug": "nautobot-welcome-wizard",
        "project_slug": "nautobot-plugin-welcome-wizard",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-welcome-wizard",
        "base_url": "nautobot-welcome-wizard",
        "min_nautobot_version": "1.5.0",
        "max_nautobot_version": "1.9999",
        "camel_name": "NautobotWelcomeWizard",
        "project_short_description": "Welcome Wizard",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/welcome-wizard/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "develop"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "setup-cookie-welcome-wizard.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

**Comment by snaselj** [comment]
Ready for review now.

cc.: @whitej6

File: docs/dev/arch_decision.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Architecture Decision Records
+
+The intention is to document deviations from a standard Model View Controller (MVC) design.
+
+!!! warning "Developer Note - Remove Me!"
```
Comment (whitej6) [inline docs/dev/arch_decision.md:5]: fill in or remove

File: docs/user/external_interactions.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# External Interactions
+
+This document describes external dependencies and prerequisites for this App to operate, including system requirements, API endpoints, interconnection or integrations to other applications or services, and similar topics.
+
+!!! warning "Developer Note - Remove Me!"
+    Optional page, remove if not applicable.
+
+## External System Integrations
```
Comment (whitej6) [inline docs/user/external_interactions.md:8]: fill in or remove

**Review by whitej6 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ about: Report a reproducible bug in the current release of nautobot-welcome-wiza
 ---
 
 ### Environment
-* Python version:  <!-- Example: 3.7.7 -->
-* Nautobot version:  <!-- Example: 1.0.0 -->
+* Python version:  <!-- Example: 3.11.4 -->
+* Nautobot version:  <!-- Example: 1.6.0 -->
```
Comment (jvanderaa) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: Not really necessary to update.

File: .flake8 [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [flake8]
-# E501: Line length is enforced by Black, so flake8 doesn't need to check it
-# W503: Black disagrees with this rule, as does PEP 8; Black wins
-ignore = E501, W503
+ignore =
+  E501,  # Line length is enforced by Black, so flake8 doesn't need to check it
+  W503   # Black disagrees with this rule, as does PEP 8; Black wins
```
Comment (jvanderaa) [inline .flake8:4]: What is this changing? Doesn't seem to have a functional change.

File: .gitignore [hunk 5]
```diff
@@ -303,8 +303,7 @@ development/*.txt
 invoke.yml
 
 # Docs
-docs/README.md
-docs/CHANGELOG.md
```
Comment (jvanderaa) [inline .gitignore:307]: Why are we removing these?

File: development/docker-compose.base.yml [hunk 6]
```diff
@@ -21,8 +21,9 @@ services:
         condition: "service_started"
       db:
         condition: "service_healthy"
-    <<: *nautobot-build
-    <<: *nautobot-base
+    <<:
+      - *nautobot-base
+      - *nautobot-build
```
Comment (jvanderaa) [inline development/docker-compose.base.yml:26]:
```suggestion
    <<: =[*nautobot-base, *nautobot-build]
```
Suggestion to follow what has come before.

**Review by jvanderaa — CHANGES_REQUESTED** [review]

File: development/development.env [hunk 7]
```diff
@@ -7,8 +7,6 @@ NAUTOBOT_BANNER_TOP="Local"
 NAUTOBOT_CHANGELOG_RETENTION=0
 
 NAUTOBOT_DEBUG=True
-NAUTOBOT_DJANGO_EXTENSIONS_ENABLED=True
-NAUTOBOT_DJANGO_TOOLBAR_ENABLED=True
```
Comment (jvanderaa) [inline development/development.env:11]: What is with removing these?

File: development/docker-compose.dev.yml [hunk 8]
```diff
@@ -12,6 +12,8 @@ services:
     volumes:
       - "./nautobot_config.py:/opt/nautobot/nautobot_config.py"
       - "../:/source"
+    healthcheck:
+      test: ["CMD", "true"]  # Due to layering, disable: true won't work. Instead, change the test
```
Comment (jvanderaa) [inline development/docker-compose.dev.yml:16]: Remove.

File: development/docker-compose.dev.yml [hunk 9]
```diff
@@ -30,6 +32,8 @@ services:
...
```
Comment (jvanderaa) [inline development/docker-compose.dev.yml:36]: Remove.

**Review by jvanderaa — CHANGES_REQUESTED** [review]

**Review by jvanderaa — APPROVED** [review]

**Review by whitej6 — APPROVED** [review]

**Pull request merged**